src/cmdline.ggo

```diff
@@ -14,7 +14,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 package "u2f-host"
-purpose "Perform U2F host-side operations on the command line. Reads challenge from standard input and writes a response to standard output."
+purpose "Perform U2F host-side operations on the command-line. Reads challenge from standard input and writes a response to standard output."
```

Comment:
My understanding is that this usage of "command line" is correct since it's the object of this sentence, not a modifier. Opinions?